public/app/features/alerting/unified/utils/rules.ts

```diff
@@ -71,6 +72,34 @@ export function alertStateToReadable(state: PromAlertingRuleState | GrafanaAlert
   return capitalize(state);
 }
 
+export function LegacyAlertStateToGrafanaAlertState(legacyState: AlertState): GrafanaAlertState {
```

Comment:
Not too sure about this function; I did not find anything similar but the annotations still use the older legacy `AlertState` states.